docs/source/getting_started/index.rst

```diff
@@ -0,0 +1,12 @@
+===============
+Getting Started
+===============
+
+.. toctree::
+   :maxdepth: 1
+
+
+   introduction
+   installation
+   ordered_logit_example.ipynb
```

Comment:
the ordered logit example seems to not have been committed yet. Possibly, because of .gitignore?

Reply:
Done.